src/main/sensors/battery.h

```diff
@@ -27,6 +27,7 @@
 typedef struct batteryConfig_s {
     // voltage
     uint8_t vbatmaxcellvoltage;             // maximum voltage per cell, used for auto-detecting battery voltage in 0.1V units, default is 43 (4.3V)
+    uint8_t vbatfullcellvoltage;            // Cell voltage at which the battery is deemed to be "full" 0.1V units, default is 41 (4.1V)
```

Comment:
new settings should be added to the END of a PG declaration otherwise the PG version needs bumping